scripts/cli/src/tests/15_itn_rewards.ts

```diff
@@ -106,7 +107,9 @@ async function main(): Promise<void> {
     )
   );
 
-  
+  console.log("ItnRewards: Dave gets Itn Rewards");
+  await claimItnReward(alice, dave.publicKey);
```

Comment:
```suggestion
  await claimItnReward(alice, dave);
```
Just pass in `dave` so you can use it for signing the claim.